mqtt/Dockerfile.armv7

```diff
@@ -0,0 +1,30 @@
+# -*- mode: dockerfile -*-
+#
+# An example Dockerfile showing how to build a Rust executable using this
+# image, and deploy it with a tiny Alpine Linux container.
+
+# You can override this `--build-arg BASE_IMAGE=...` to use different
+# version of Rust or OpenSSL.
+ARG BASE_IMAGE=messense/rust-musl-cross:armv7-musleabihf
```

Comment:
Could you consult with @arsing to resolve an issue you face with backtrace-sys crate for arm32 musl build?